content/shell/android/shell_apk/src/org/chromium/content_shell_apk/ContentShellActivity.java

```diff
@@ -21,8 +21,8 @@
 import org.chromium.content_public.browser.WebContents;
 import org.chromium.content_shell.Shell;
 import org.chromium.content_shell.ShellManager;
-import org.chromium.ui.base.ActivityWindowAndroid;
-import org.chromium.ui.base.IntentRequestTracker;
+// import org.chromium.ui.base.ActivityWindowAndroid;
```

Comment:
Why are any of those changes needed in ContentShellActivity ? We cloned it to CobaltActivity already - didnt we ?